hack/run-ci-e2e-test.sh

```diff
@@ -115,8 +123,10 @@ done
 # -flag x is allowed for non-boolean flags only(https://golang.org/pkg/flag/)
 
 # Test that the operator is running when the private key secret is not present
-printf "\n####### Testing operator deployed without private key secret #######\n" >> "$ARTIFACT_DIR"/wmco.log
-go test ./test/e2e/... -run=TestWMCO/operator_deployed_without_private_key_secret -v -args --node-count=$NODE_COUNT --private-key-path=$KUBE_SSH_KEY_PATH $WMCO_PATH_OPTION
+if [[ "$TEST" = "all" || "$TEST" = "basic" ]]; then
+  printf "\n####### Testing operator deployed without private key secret #######\n" >> "$ARTIFACT_DIR"/wmco.log
+  go test ./test/e2e/... -run=TestWMCO/operator_deployed_without_private_key_secret -v -args --node-count=$NODE_COUNT --private-key-path=$KUBE_SSH_KEY_PATH $WMCO_PATH_OPTION
+fi
```

Comment:
This test is just checking if the operator is deployed. It should complete almost instantly. I'm not sure that there is a reason we wouldnt want it to run

Reply:
OK, the name of the test made think this was a special case. I will update it to run always.